src/drivers/imu/bosch/bmi088/BMI088_Accelerometer.cpp

```diff
@@ -286,7 +285,7 @@ void BMI088_Accelerometer::ConfigureAccel()
 
 void BMI088_Accelerometer::ConfigureSampleRate(int sample_rate)
 {
-	if (sample_rate == 0 || sample_rate > 800) {
```

Comment:
Not strictly required, but I just added this to avoid unnecessary load for higher rates.